test/testenv/appframework_utils.go

```diff
@@ -46,6 +46,10 @@ func GenerateAppSourceSpec(appSourceName string, appSourceLocation string, appSo
 
 // GetPodAppStatus Get the app install status and version number
 func GetPodAppStatus(deployment *Deployment, podName string, ns string, appname string, clusterWideInstall bool) (string, string, error) {
+	// For clusterwide install do not check for versions on deployer and cluster-master as the apps arent installed there
+	if clusterWideInstall && (strings.Contains(podName, "-cluster-master-") || strings.Contains(podName, "-deployer-")) {
```

Comment:
Don't we explicitly confirm that in a cluster wide install, apps are indeed not installed on cm / deployer?

Reply:
Since if an app is not present/installed. The command `./splunk display app 'appname'`would return an invalid command exit code 22.

So instead we check that the app was not downloaded by checking that there is no folder in respect to the app.